src/main/java/com/epam/aidial/core/log/GfLogStore.java

```diff
@@ -77,7 +78,8 @@ private void append(ProxyContext context, LogEntry entry) throws JsonProcessingE
         append(entry, request.getHeader(Proxy.HEADER_JOB_TITLE), true);
         append(entry, "\"}", false);
 
-        TokenUsage tokenUsage = context.getTokenUsage();
+        DeploymentCostStats deploymentCostStats = context.getDeploymentCostStats();
+        TokenUsage tokenUsage = deploymentCostStats.getTokenUsage();
```

Comment:
this may lead to NPE in DeploymentFeatureController

Reply:
fixed